testgrid/specs/storage-migration.yaml

```diff
@@ -0,0 +1,767 @@
+- name: Migrate from Rook 1.0.4 to OpenEBS + Minio
```

Comment:
We have none tests with docker and unfortunately it still used.

Can we add a test with the migration that we have been facing issues
(Migration from Rook with Docker config and k8s 1.19 to 1.21)

install:
```
spec:
  kubernetes:
    version: 1.19.3
  docker:
    daemonConfig: |
      {
        \"exec-opts\": [\"native.cgroupdriver=systemd\"],
        \"log-opts\": {
            \"max-size\": \"100m\",
            \"max-file\": \"3\"
            }
      }
    version: 19.03.10
  weave:
    version: 2.6.5
  rook:
    version: 1.0.4
  openebs:
    isCstorEnabled: false
    isLocalPVEnabled: true
    localPVStorageClassName: openebs
    namespace: openebs
    version: 1.6.0
  contour:
    tlsMinimumProtocolVersion: '1.2'
    version: 1.22.0
  registry:
    version: 2.7.1
  prometheus:
    version: 0.58.0-39.4.0
  kotsadm:
    version: 1.57.0
  velero:
    version: 1.9.0
  ekco:
    minReadyMasterNodeCount: 2
    minReadyWorkerNodeCount: 0
    nodeUnreachableToleration: 5m
    rookShouldUseAllNodes: false
    shouldDisableClearNodes: false
    shouldDisableRebootServices: false
    shouldEnablePurgeNodes: false
    version: 0.16.0
  goldpinger:
    version: 3.2.0-4.1.1

```

upgrade 


```
spec:
  kubernetes:
    version: 1.21.14
  docker:
    daemonConfig: |
      {
        \"exec-opts\": [\"native.cgroupdriver=systemd\"],
        \"log-opts\": {
          \"max-size\": \"100m\",
          \"max-file\": \"3\"
        }
      }
    version: 19.03.10
  weave:
    version: 2.6.5-20221025
  openebs:
    isCstorEnabled: false
    isLocalPVEnabled: true
    localPVStorageClassName: openebs
    namespace: openebs
    version: 3.3.0
  minio:
    version: 2022-10-20T00-55-09Z
  contour:
    tlsMinimumProtocolVersion: '1.2'
    version: 1.23.0
  registry:
    version: 2.8.1
  kotsadm:
    version: 1.90.0
    s3Override: https://s3.kurl.sh/external/kotsadm-1.90.0.tar.gz
  velero:
    resticTimeout: 24h0m0s
    version: 1.9.2
  ekco:
    minReadyMasterNodeCount: 2
    minReadyWorkerNodeCount: 0
    nodeUnreachableToleration: 5m
    shouldDisableClearNodes: false
    shouldDisableRebootServices: false
    shouldEnablePurgeNodes: false
    version: latest
  goldpinger:
    version: 3.7.0-5.5.0
```

Reply:
Great idea. I have added a test case using Docker instead of containerd.